app/src/main/java/com/bnyro/recorder/services/AudioRecorderService.kt

```diff
@@ -24,15 +24,16 @@ class AudioRecorderService : RecorderService() {
             ).let {
                 setAudioSource(it)
             }
-            if (audioFormat.codec != MediaRecorder.AudioEncoder.OPUS) {
-                Preferences.prefs.getInt(Preferences.audioSampleRateKey, -1).takeIf { it > 0 }
-                    ?.let {
+            Preferences.prefs.getInt(Preferences.audioSampleRateKey, -1).takeIf { it > 0 }
```

Comment:
We should extract this and the bitrate preference to variables first, that would make things cleaner.

Reply:
Bitrate preference is exactly as all the other preferences even before the issue with opus recording was discovered so I haven't yet touched that one. If you insist I will change that of course.
As for the sample rate I have extracted it to a variable but I am not sure it's the way you would like to have it. I did want to introduce more conditions so the change is just minimal.